src/instructlab/model/train.py

```diff
@@ -1095,42 +1114,42 @@ def _run_phased_training(
 
         phase_model.ended_at_utc = TrainingJournal.now_utc()
 
-        journal.current_phase = TrainingPhases.EVAL1
+        journal.current_phase = TrainingPhases.TRAIN2
         journal.commit()
 
         logger.debug("Finished training #1\n%s", journal.print_model_rich())
     else:
         click.secho("SKIPPING: Training Phase 1/2; already in Journal", fg="cyan")
 
-    if journal.current_phase == TrainingPhases.EVAL1:
-        click.secho("MMLU evaluation for Phase 1...", fg="cyan")
+    # if journal.current_phase == TrainingPhases.EVAL1:
```

Comment:
this hurts my soul a bit but I understand we want to keep this for future purposes

Reply:
yeah I don't like it either